.github/workflows/pull-request.yml

```
@@ -449,3 +449,147 @@
+          echo "# API Reference
+          
+          This section contains the auto-generated API documentation for Talawa Admin.
+          " > docs/docs/developer-guide/reference/README.md
+
+      - name: Setup Git Config
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+      
+      - name: Debug GitHub Variables
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)